packages/controls/src/controls/shape-v2/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './shape'
```

Comment:
This commit will be "fixed down" into the commit below before I merge, just wanted to make reviewing a bit easier so you can compare the two Shape controls